readme.md

```diff
@@ -92,7 +96,9 @@ Using Ruby, instantiate an array called `fruits` that contains `apple`, `banana`
 Then, use an enumerator to print to the console the sentence "I'd like to eat a [fruit]" once for each fruit.
 
 ```rb
-# Your answer...
+var @fruits = ["apple", "banana", "orange"]
+
+console.log(@fruits.each)
```

Comment:
Remember that to use `.each` we have to provide a block of code to tell it what to do (also, we can't use `console.log` because we're in ruby, not JS):


```ruby
fruits = ["apple", "banana", "orange"]
fruits.each do |fruit|
  puts "I'd like to eat a #{fruit}"
end
```